### DISCUSSION TITLE
XP12 Airfoil Updates

### DISCUSSION THREAD

**Discussion by remenkemi** [Stratosphere-Studios/777-300ER#50] [General]
@zeta976 Can you check the airfoils in the latest airfoil maker version? Austin made some changes to how they work.
https://www.youtube.com/watch?v=Ruh93CwQBCI

**Comment by BRUHegg** [comment]
> @zeta976 Can you check the airfoils in the latest airfoil maker version? Austin made some changes to how they work. https://www.youtube.com/watch?v=Ruh93CwQBCI

It should be fine. TLDR of the video: he messed up the critical mach number in rc1 and rc2 and fixed it in rc3. So, theoretically, it should fly just like it did before rc1.

**Reply by remenkemi** [reply]
Timestamp? From what I understood this is new. I'm also curious as to how close X-Planes estimated critical mach number and the one Zeta used are. We also might want to look into the area rule stuff.

**Reply by zeta976** [reply]
In the last update to the airfoils I corrected the critical mach number and the thickness of every airfoil so those values should be right. 

The video talks about other parameters that we should tweak in plane maker:

- The 777 does not appear to be strictly area-ruled so a value of around 0.75 should be acceptable. 
- The body coefficient of drag should be somewhere around 0.08 based on my calculations.
- Enter an inlet compression efficiency of 0.95
- Make sure to enter the right wing sweep value.
- Tweak the flap cl, cd, and cm values to get the right low-speed performance.

**Reply by remenkemi** [reply]
Ok. I'm curious as to how close xplane's estimate is to the values you used to see how accurate it is. If you cant get planemaker running I can put in those values for you.

**Comment by remenkemi** [comment]
> In the last update to the airfoils I corrected the critical mach number and the thickness of every airfoil so those values should be right.
> 
> The video talks about other parameters that we should tweak in plane maker:
> 
>     * The 777 does not appear to be strictly area-ruled so a value of around 0.75 should be acceptable.
> 
>     * The body coefficient of drag should be somewhere around 0.08 based on my calculations.
> 
>     * Enter an inlet compression efficiency of 0.95
> 
>     * Make sure to enter the right wing sweep value.
> 
>     * Tweak the flap cl, cd, and cm values to get the right low-speed performance.
ok so @zeta976
     - where is that setting?
     - Looks like we have .084 set. Planemaker suggests .071
     - Already set
     - I think this is already correct, I did research on it a long time ago when I was still working on the flight model.
     - @BRUHegg has been taking care of those values

**Reply by BRUHegg** [reply]
@crazytimtimtim you can close this issue. These things have already been addressed.
